sidebarsDocs.js

```diff
@@ -139,8 +139,18 @@ module.exports = {
       link: { type: 'doc', id: 'network-iot/lorawan-network-servers/lorawan-network-servers' },
       items: [
         'network-iot/lorawan-network-servers/find-a-lns-provider',
-        'network-iot/lorawan-network-servers/run-a-lns',
-        'network-iot/buy-an-oui',
+        {
+          type: 'category',
+          label: 'Run Your Own LNS',
+          link: { type: 'doc', id: 'network-iot/run-an-lns/run-an-lns' },
+          items: [
+            'network-iot/run-an-lns/buy-an-oui',
+            'network-iot/run-an-lns/fund-an-oui',
+            'network-iot/run-an-lns/configure-chirpstack',
+            'network-iot/run-an-lns/configure-routing-rules',
+            'network-iot/run-an-lns/advanced-configuration',
```

Comment:
maybe add a trouble shooting section with common issues

- no incoming data => check route, deveuis
- join received but nothing else => check skf, check max_copies, check rssi 

+ a reference to where they can go for other questions / support